apps/vanilla-three/src/main.js

```diff
@@ -0,0 +1,27 @@
+import * as THREE from '@jscadui/bundle-three'
+import { RenderThree } from '@jscadui/render-three'
+import { makeAxes, makeGrid } from '@jscadui/scene'
+import { light } from '@jscadui/themes'
```

Comment:
`import { light as theme } from '@jscadui/themes'` if necessary to use theme naming I rather give it in here.